modules/opus/SCsub

```diff
@@ -220,6 +220,12 @@ if env['builtin_opus']:
     ]
     env_opus.Append(CPPPATH=[thirdparty_dir + "/" + dir for dir in thirdparty_include_paths])
 
+    if env["platform"] == "android" or env["platform"] == "iphone":
+        if ("arch" in env and env["arch"] == "arm") or ("android_arch" in env and env["android_arch"] in ["armv6", "armv7"]):
+            env_opus.Append(CFLAGS=["-DOPUS_ARM_OPT"])
+        elif ("arch" in env and env["arch"] == "arm64") or ("android_arch" in env and env["android_arch"] == "arm64v8"):
```

Comment:
The only actual logic change is here: now we're also enabling `OPUS_ARM64_OPT` on `arm64v8` for Android. Previously it has been left out.